File: Domino/BoardDomino.cpp
#include "stdafx.h"
#include "BoardDomino.h"


BoardDomino::BoardDomino(void)
{
}

BoardDomino::~BoardDomino(void)
{
}



BlockDomino BoardDomino::get_blockdomino_jackpot(int at)
{
	return (blockdomino_jackpot.at(at));
}
BlockDomino BoardDomino::get_blockdomino_player1(int at)
{
	return blockdomino_player1.at(at);
}
BlockDomino BoardDomino::get_blockdomino_player2(int at)
{
	return blockdomino_player2.at(at);
}
BlockDomino BoardDomino::get_blockdomino_onboard(int at)
{
	return blockdomino_onboard.at(at);
}

File: Domino C#/Source/Domino/BoardDomino.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace DominoGame
{
    class BoardDomino
    {
        protected List<BlockDomino> blockdomino_jackpot;
        protected List<BlockDomino> blockdomino_player1;
        protected List<BlockDomino> blockdomino_player2;
        protected List<BlockDomino> blockdomino_onboard;

        public BoardDomino()
        {
            blockdomino_jackpot = new List<BlockDomino>();
            blockdomino_player1 = new List<BlockDomino>();
            blockdomino_player2 = new List<BlockDomino>();
            blockdomino_onboard = new List<BlockDomino>();

            
        }
    }
}
